src/views/Recorder/EmptyState.tsx

```diff
@@ -85,48 +112,97 @@ export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
           margin-top: var(--space-6);
         `}
       >
-        <FieldGroup
-          name="url"
-          label="Starting URL"
-          hint="Enter the URL of the website or service you want to test"
-          hintType="text"
-          errors={errors}
-          width="100%"
-        >
-          <Flex>
-            <TextField.Root
-              {...register('url')}
-              placeholder="e.g. quickpizza.grafana.com"
-              autoFocus
-              css={css`
-                flex-grow: 1;
-                border-right: 0;
-                border-bottom-right-radius: 0;
-                border-top-right-radius: 0;
-              `}
+        <Flex direction="column" gap="4">
+          <div>
+            <FieldGroup
+              name="url"
+              label="Starting URL"
+              hint="Enter the URL of the website or service you want to test"
+              hintType="text"
+              errors={errors}
+              width="100%"
+            >
+              <Flex>
+                <TextField.Root
+                  {...register('url')}
+                  placeholder="e.g. quickpizza.grafana.com"
+                  autoFocus
+                  css={css`
+                    flex-grow: 1;
+                    border-right: 0;
+                    border-bottom-right-radius: 0;
+                    border-top-right-radius: 0;
+                  `}
+                />
+                <Button
+                  disabled={isLoading || !canRecord}
+                  type="submit"
+                  css={css`
+                    margin-left: -1px;
+                    border-bottom-left-radius: 0;
+                    border-top-left-radius: 0;
+                  `}
+                >
+                  {isLoading ? <Spinner /> : <DiscIcon />} Start recording
+                </Button>
+              </Flex>
+            </FieldGroup>
+
+            <WarningMessage
+              proxyStatus={proxyStatus}
+              isBrowserInstalled={isBrowserInstalled}
             />
-            <Button
-              disabled={isLoading || !canRecord}
-              type="submit"
-              css={css`
-                margin-left: -1px;
-                border-bottom-left-radius: 0;
-                border-top-left-radius: 0;
-              `}
+          </div>
+
+          {canCaptureBrowser && (
+            <FieldGroup
+              name="captureBrowser"
+              label={<CaptureBrowserLabel />}
+              hint="Record user interactions in the browser alongside network requests."
+              hintType="text"
             >
-              {isLoading ? <Spinner /> : <DiscIcon />} Start recording
-            </Button>
-          </Flex>
-        </FieldGroup>
-        <WarningMessage
-          proxyStatus={proxyStatus}
-          isBrowserInstalled={isBrowserInstalled}
-        />
+              <Text size="2">
+                <Flex gap="2" align="center">
+                  <Checkbox
```

Comment:
We can move the `Flex` inside the `Text` component so the label is clickable. I think this improves the UX and makes it consistent with the behaviour of other checkboxes in the app.

https://github.com/user-attachments/assets/609f30cc-10b1-46d0-9814-282ced96de65

Reply:
Good catch! It was working that way, but I was changing things around last minute and broke it. Will fix.